packages/fxa-auth-server/lib/routes/cloud-scheduler.ts

```diff
@@ -4,19 +4,96 @@
 
 import { ConfigType } from '../../config';
 import { AuthLogger, AuthRequest } from '../types';
-// commenting this out for now because the script uses p-queue, which is pure
-// ESM since 7.x.  That won't work when nodejs try to `require` it.
-// import { processDateRange } from '../../scripts/delete-unverified-accounts';
 import {
   AccountTasks,
   AccountTasksFactory,
-  //  ReasonForDeletion,
+  ReasonForDeletion,
 } from '@fxa/shared/cloud-tasks';
 import { StatsD } from 'hot-shots';
-import error from '../error';
+import { setupAccountDatabase } from '@fxa/shared/db/mysql/account';
 
 const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
 
+/**
+ * Process a date range of accounts to delete.
+ *
+ * @param config
+ * @param accountTasks
+ * @param reason
+ * @param startDate
+ * @param endDate
+ * @param taskLimit
+ */
+export async function processDateRange(
```

Comment:
This name's too generic for something that's only meant to delete accounts.

Reply:
went with `processAccountDeletionInRange`